src/tests/JIT/Performance/RunBenchmarks/RunBenchmarks.cs

```diff
@@ -24,7 +24,7 @@
 //     -listtags                  prints a list of the tag names and does nothing else
 //     -listexes                  prints a list of full path names to executables in benchmarks
 //     -runner                    run benchmarks on using runner(e.g. corerun, default is DesktopCLR)
-//     -complus_version <version> run benchmarks on particular DesktopCLR version
```

Comment:
I don't think you should change this file (especially the complus_version stuff). Actually, I think this whole directory is dead code that should be removed.

Reply:
Deleted RunBenchmarks directory.